.github/workflows/scheduled.yaml

```diff
@@ -0,0 +1,85 @@
+name: Run scheduled jobs
+
+on:
+  schedule:
+    - cron: '0 0 * * *' # Runs daily at midnight
```

Comment:
UTC midnight is 4pm on the US west coast so we might need to push to a few hours later at some point. But that's easy enough to change.